rl4co/tasks/eval.py

```diff
@@ -169,23 +169,20 @@ def __init__(self, env, samples, softmax_temp=None, **kwargs):
         self.softmax_temp = softmax_temp
 
     def _inner(self, policy, td):
-        td = batchify(td, self.samples)
         out = policy(
             td.clone(),
             decode_type="sampling",
-            num_starts=0,
+            num_starts=self.samples,
+            multistart=True,
```

Comment:
Won't this force the use of `select_start_nodes_fn`?

Reply:
great catch! Fix incoming